addons/knowledge/static/src/js/knowledge_renderers.js

```diff
@@ -108,6 +108,10 @@ const KnowledgeArticleFormRenderer = FormRenderer.extend(KnowledgeTreePanelMixin
                             $(child).data('category', data.newCategory);
                         });
                         $sortable.sortable('enable');
+                        let $info_box = $li.siblings('.o_knowledge_empty_info');
+                        if($info_box.length != 0) {
```

Comment:
Do we need this if, aka if there isn't any node is there an issue ? Can't we do directly "$li.siblings('.o_knowledge_empty_info').addClass('d-none');" ?